<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Программа-диспетчер для ОС Windows</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>771</width>
      <height>521</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabProcesses">
     <attribute name="title">
      <string>Процессы</string>
     </attribute>
     <widget class="QTableView" name="tableViewProcesses">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>741</width>
        <height>311</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonKillProcess">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>330</y>
        <width>128</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Завершить процесс</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBoxVirtualMemoryManagement">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>360</y>
        <width>741</width>
        <height>121</height>
       </rect>
      </property>
      <property name="title">
       <string>Выделение виртуальной памяти выбранному выше процессу:</string>
      </property>
      <widget class="QLabel" name="labelDataFromMemory">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>80</y>
         <width>561</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonWriteToMemory">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>50</y>
         <width>128</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Поместить в память</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonFreeMemory">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>20</y>
         <width>128</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Освободить память</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditUserDataForMemory">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>571</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Здравствуй, мир!</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBoxMemorySize">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>128</width>
         <height>22</height>
        </rect>
       </property>
       <property name="suffix">
        <string/>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
       <property name="value">
        <number>1024</number>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonReadFromMemory">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>128</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Считать из памяти</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonAllocMemory">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>20</y>
         <width>128</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Выделить память</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>155</x>
         <y>22</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>KB</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tabPerformance">
     <attribute name="title">
      <string>Быстродействие</string>
     </attribute>
     <widget class="QGroupBox" name="groupBoxPhysicalMemory">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>271</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>Физическая память (МБ)</string>
      </property>
      <widget class="QLabel" name="labelPhysicalMemoryTotalTitle">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Всего</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelPhysicalMemoryTotal">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>30</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="labelPhysicalMemoryAvailableTitle">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Доступно</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelPhysicalMemoryAvailable">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>60</y>
         <width>161</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionFileNewProcess"/>
    <addaction name="separator"/>
    <addaction name="actionFileClose"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Параметры</string>
    </property>
    <addaction name="actionAlwaysOnTop"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>Вид</string>
    </property>
    <addaction name="action_ViewRefresh"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="actionHelpAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionFileClose">
   <property name="text">
    <string>Завершение программы-диспетчера</string>
   </property>
  </action>
  <action name="actionHelpAbout">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="action_ViewRefresh">
   <property name="text">
    <string>Обновить</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionFileNewProcess">
   <property name="text">
    <string>Новый процесс</string>
   </property>
  </action>
  <action name="actionAlwaysOnTop">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Поверх остальных окон</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
